src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -41,11 +44,23 @@ public AddCommand(String name,
                       String phone, boolean isPhonePrivate,
                       String email, boolean isEmailPrivate,
                       String address, boolean isAddressPrivate,
+                      String postalCode, boolean isPostalCodePrivate,
                       Set<String> tags) throws IllegalValueException {
         final Set<Tag> tagSet = new HashSet<>();
         for (String tagName : tags) {
             tagSet.add(new Tag(tagName));
         }
+
+        PostalCode newPostalCode = new PostalCode(postalCode, isPostalCodePrivate);
```

Comment:
This constructor throws an exception for certain values of postalCode. Should it be in a try-catch block?